src/main/java/co/omise/models/Account.java

```diff
@@ -21,7 +21,7 @@ public void setEmail(String email) {
     }
 
     /**
-     * The {@link RequestBuilder<Account>} class for for getting the user's Account. This class only overrides the path() method from its parent.
+     * The {@link RequestBuilder} class for for getting the user's Account. This class only overrides the path() method from its parent.
```

Comment:
`for for`? 😅

Reply:
I don't know why this keeps happening to me 😂